src/components/frame/Extensions/Header/hooks.ts

```diff
@@ -21,38 +18,68 @@ export const useHeader = () => {
     return firstWatchTime - secondWatchTime;
   };
 
-  const ref = useDetectClickOutside({ onTriggered: () => setIsOpen(false) });
-  const { getTooltipProps, setTooltipRef, setTriggerRef } = usePopperTooltip(
+  const popperTooltipOffset = !isMobile ? [120, 8] : [0, 20];
+  const menuPopperTooltipOffset = [0, 20];
+
+  const { getTooltipProps, setTooltipRef, setTriggerRef, visible } =
+    usePopperTooltip(
+      {
+        delayShow: 200,
```

Comment:
add delayHide: 200 if you use an interactive prop 
https://www.npmjs.com/package/react-popper-tooltip
![image](https://github.com/JoinColony/colonyCDapp/assets/119701775/fe7e63d7-09a0-4d44-a57d-b409c41b162d)

Reply:
delayHide added